test/calculator_test.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:my_books/calculator.dart';
+
+void main() {
+  late Calculator calculator;
+
+  setUp(() {
+    calculator = Calculator();
+  });
+
+  test('Sum_Test', () {
```

Comment:
you can write just 'sum test', 'division test' etc. Please correct

Reply:
done